media-libs/harfbuzz/harfbuzz-9.0.0.recipe

```diff
@@ -31,7 +31,7 @@ if [ "$targetArchitecture" = x86_gcc2 ]; then
 	commandBinDir=$prefix/bin
 fi
 
-libVersion="0.60830.0"
+libVersion="0.60900.0"
 libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
```

Comment:
General question: is this backwards compatible with the old version? or does other software that uses the libraries need to be rebuilt?

According to this it should be compatible, but the major version increase may be an indicator otherwise.

Reply:
I haven't instantly noticed things crashing or failing to link, how would I best check this?

Reply:
Ok, should be good then I think.